autonomy/cli/hash.py

```diff
@@ -55,6 +65,79 @@ def load_configuration(
     return cast(PackageConfiguration, configuration_obj)
 
 
+def update_hashes(  # pylint: disable=too-many-locals
```

Comment:
That's what I said, https://github.com/valory-xyz/open-autonomy/pull/1007#discussion_r905763076 😅. And that's why I asked you where should I update the hash command to facilitate the agent hashing. You didn't respond so I thought keeping it here should be the right thing to do since we want to keep all of the service component related stuff on open-autonomy

Reply:
Ok